packages/polymath-issuer/src/components/App.js

```diff
@@ -91,7 +98,9 @@ class App extends Component<Props> {
         <TxModal />
         <EnterPINModal />
         <ConfirmModal />
-        <AuthWrapper>{renderRoutes(route.routes)}</AuthWrapper>
+        <AuthWrapper onFail={this.onAuthFail.bind(this)}>
```

Comment:
Mind if we agree on not use bind and use class properties instead?

```js
class Foo {
  onAuthFail = () => {} // This will be binded

  render() {
    // code
    return <Foo onFail={this.onFail} />
  }
}
```